keystore/filesystem/internal/msgp_structs.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016, Cossack Labs Limited
```

Comment:
Shouldn't we put current year here?
```suggestion
// Copyright 2022, Cossack Labs Limited
```